### ISSUE TITLE
Make multi-row "simple" footer work with responsive

### ISSUE THREAD

**Issue by exteon** [DataTables/DataTablesSrc#218]
I understand that, as per https://datatables.net/forums/discussion/63149/colspan-header-responsive-issue, "complex" headers/footers are not supported. However, a multi-row footer without `colspan` or `rowspan` would hardly qualify as "complex" and is actually quite common (i.e. I'm using to display 2 rows containing filtered totals and unfiltered totals for each column). The responsive plugin works oly for the first row of the footer, which is inconsistent anyway (either work for all, or for none, but I get different length rows).

**Comment by AllanJard** [comment]
Yes you are right, that isn't really complex. Support for that will fall out of the work for colspan / rowspan.

**Comment by AllanJard** [comment]
Complex headers and footers [will work with Responsive](https://datatables.net/extensions/responsive/examples/initialisation/complex-header.html). I forgot to close this off at the time - sorry.
